addons/mail/static/src/models/attachment/attachment.js

```diff
@@ -393,6 +409,9 @@ function factory(dependencies) {
         displayName: attr({
             compute: '_computeDisplayName',
         }),
+        displaySize: attr({
+            compute: '_computeDisplaySize',
+        }),
```

Comment:
Missing documentation on this field.

Usually fields that are prefixed with "display" are poorly named and/or current design of model and fields is poor.

At a glance, the intent is to display a human-readable format of the size. I would name it to something like this without the "display" part (as it's inappropriately too specific, we might use this for other means than just display).